Moskalenkov-homework-2.cpp

```diff
@@ -0,0 +1,153 @@
+#include <iostream>
+
+int main() {
+    std::cout << "What task do you want to solve ?" << std::endl;
+
+    int setNumber(0);
+
+    while (setNumber < 1 || setNumber > 5) {
+        std::cout
+            << "1: Calculate the sum and arithmetic mean of digits in the number"
+            << std::endl;
+        std::cout << "2: Lucky ticket " << std::endl;
+        std::cout << "3: Inverse numbers " << std::endl;
+        std::cout << "4: Sum of odd elements " << std::endl;
+        std::cout << "5: Best divisor " << std::endl;
+        std::cin >> setNumber;
+    }
+    if (setNumber == 1) {
+        int number(0);
+        int sum(0);
+        int i(0);
```

Comment:
настоятельно рекомендую использовать универсальную инициализацию
```suggestion
        int number{};
        int sum{};
        int i{};
```